ngrok/toggle-ngrok.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# toggle-ngrok.sh
+#
+# Flips the enabled field on the configmap, and updates
+# the button text to reflect the new state.
+
+set -euo pipefail
+
+name="$1"
+child_name="ngrok:$name"
+cm="$(tilt get configmap "$child_name" -o json)"
+enabled="$(echo "$cm" | jq -r ".data.enabled")"
+new_val="true"
+new_text="stop ngrok"
+if [[ "$enabled" == "true" ]]; then
+    new_val="false"
+    new_text="start ngrok"
+fi
+
+tilt patch uibutton "$child_name" -p "{\"spec\":{\"text\":\"$new_text\"}}"
```

Comment:
might be worth also setting its icon to play/stop - on table view, the text only appears as a tooltip

Reply:
fixed!